hypervisor/dm/vpci/vpci.c

```diff
@@ -483,10 +483,24 @@ static int32_t write_cfg(struct acrn_vpci *vpci, union pci_bdf bdf,
 }
 
 /**
+ * @brief Initialize a vdev structure.
+ *
+ * The function vpci_init_vdev is used to initialize a vdev structure with a PCI device configuration(dev_config)
+ * on a specified vPCI bus(vpci). If the function vpci_init_vdev initializes a SRIOV Virtual Function(VF) vdev structure,
+ * the parameter pf_vdev is the VF associated Physical Function(PF) vdev structure, otherwise the parameter pf_vdev is NULL.
+ * The caller of the function vpci_init_vdev should guarantee execution atomically.
+ *
+ * @param vpci       Pointer to a vpci data structure
+ * @param dev_config Pointer to a dev_config structure
```

Comment:
def_config actually contains the vdev to operate, right?
So pointer to a dev_config structure of the vdev

Reply:
OK